client/Makefile

```diff
@@ -9,17 +9,16 @@ include ../common/Makefile.common
 CC=gcc
 CXX=g++
 #COMMON_FLAGS = -m32
-VPATH = ../common ../zlib
+VPATH = ../common ../zlib ../tools
 OBJDIR = obj
 
 LDLIBS = -L/opt/local/lib -L/usr/local/lib -lreadline -lpthread -lm
 LUALIB = ../liblua/liblua.a
 LDFLAGS = $(COMMON_FLAGS)
-CFLAGS = -std=c99 -I. -I../include -I../common -I../zlib -I/opt/local/include -I../liblua -Wall $(COMMON_FLAGS) -g -O4
+CFLAGS = -std=c99 -D_ISOC99_SOURCE -I. -I../include -I../common -I../tools -I../zlib -I/opt/local/include -I../liblua -Wall $(COMMON_FLAGS) -g -O4
 LUAPLATFORM = generic
 
 ifneq (,$(findstring MINGW,$(platform)))
-    CFLAGS += -D__USE_MINGW_ANSI_STDIO=1
     CXXFLAGS = -I$(QTDIR)/include -I$(QTDIR)/include/QtCore -I$(QTDIR)/include/QtGui
```

Comment:
without this one environment still complains about unknown print characters
`flasher.c: In function 'cmd_debug':
flasher.c:27:3: warning: unknown conversion type character 'z' in format [-Wformat=]
   printf("UsbCommand length[len=%zd]\n",sizeof(UsbCommand));
   ^
flasher.c:27:3: warning: too many arguments for format [-Wformat-extra-args]
flasher.c:28:3: warning: unknown conversion type character 'z' in format [-Wformat=]
   printf("  cmd[len=%zd]: %016"llx"\n",sizeof(UC->cmd),UC->cmd);
   ^`